assets/helpers/contributions.js

```diff
@@ -0,0 +1,22 @@
+// @flow
```

Comment:
Ok, we should revisit the discussion about having a centralised place for types, I think it would make the code more readable.